scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -61,12 +55,32 @@ import kotlinx.coroutines.asCoroutineDispatcher
 import kotlinx.coroutines.async
 import kotlinx.coroutines.coroutineScope
 import kotlinx.coroutines.withContext
+import java.security.MessageDigest
 
 /**
  * Implementation of [Scanner] for scanners that operate locally. Packages passed to [scanPackages] are processed in
  * serial order. Scan results can be stored in a [ScanResultsStorage].
  */
 abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanner(name, config), CommandLineTool {
+    companion object {
+        val DEFAULT_ARCHIVE_DIR by lazy { getUserOrtDirectory().resolve("scanner/archive") }
+
+        val DEFAULT_ARCHIVE_PATTERNS = listOf(
```

Comment:
Hmm, this basically duplicates the list in `FileMatcher`. Looks to be like we should actually keep the global `LICENSE_FILE_NAMES` that was removed in a0d12dfa91b37650b8b8b78d277d690757266ed2 and use it here as well as in `FileMatcher`.

Reply:
It's not exactly the same list, but I agree it makes sense to have the common part in a single place. I would address this in a separate PR if that is ok for you.

Reply:
Yes, that's fine!